robolectric/src/main/java/org/robolectric/Robolectric.java

```diff
@@ -117,7 +118,8 @@ public static <T extends Activity> ActivityController<T> buildActivity(
    * @deprecated use {@link androidx.test.core.app.ActivityScenario}
    */
   @Deprecated
-  public static <T extends Activity> T setupActivity(Class<T> activityClass) {
+  @SuppressWarnings("InlineMeSuggester")
+  public static final <T extends Activity> T setupActivity(Class<T> activityClass) {
```

Comment:
if I add `@Inline` annotation here, the Gradle build script takes forever to finish. Don't know the reason, trying to figure out as well.

Reply:
We don't need to add `InlineMe` for this method, because this method is on the plan to remove `@Deprecated`.